docs/conceptual/gpu-arch.md

```diff
@@ -5,11 +5,20 @@
   MI100, AMD Instinct">
 </head>
 
+(gpu-arch-documentation)=
+
 # GPU architecture documentation
 
 :::::{grid} 1 1 2 2
 :gutter: 1
 
+:::{grid-item-card}
```

Comment:
I am not sure if we should create a generic grid item. It would be better just list the generic things under the grid.

Reply:
Moved the table.